ocs_ci/ocs/ui/helpers_ui.py

```diff
@@ -95,6 +99,156 @@ def format_locator(locator, string_to_insert):
     return locator[0].format(string_to_insert), locator[1]
 
 
+def create_storage_class_ui(
+    setup_ui,
+    encryption=False,
+    backend_path=None,
+    vault_namespace=None,
+    reclaim_policy="Delete",
+    provisioner="rbd",
+    vol_binding_mode="WaitForFirstConsumer",
```

Comment:
It would be better to specify the default volume binding mode as "Immediate" because IIRC we don't use "WaitForFirstConsumer" in OCS/ODF storageclasses

Reply:
updated